MapboxNavigationTests/CarPlayManagerTests.swift

```diff
@@ -199,7 +199,7 @@ class CarPlayManagerTests: XCTestCase {
             CLLocationCoordinate2D(latitude: 37.764793, longitude: -122.463161),
             CLLocationCoordinate2D(latitude: 34.054081, longitude: -118.243412),
         ])
-        choice.userInfo = (Fixture.route(from: "route-with-banner-instructions", options: options), options)
+        choice.userInfo = (Fixture.route(from: "route-with-banner-instructions", options: options), 0, options)
```

Comment:
The route index is almost always 0 in the unit tests and integration tests, because `Fixture.route(from:options:)` always returns the first route, and few of the fixtures come with multiple routes anyways.